src/test/app/Escrow_test.cpp

```diff
@@ -1508,6 +1508,164 @@ struct Escrow_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testCredentials()
+    {
+        testcase("Test with credentials");
+
+        using namespace jtx;
+        using namespace std::chrono;
+
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        Account const carol{"carol"};
+        Account const dillon{"dillon "};
+        Account const zelda{"zelda"};
+
+        const char credType[] = "abcde";
+
+        {
+            // Credentials amendment not enabled
+            Env env(*this, supported_amendments() - featureCredentials);
+            env.fund(XRP(5000), alice, bob);
+            env.close();
+
+            auto const seq2 = env.seq(alice);
+            env(escrow(alice, bob, XRP(1000)),
+                finish_time(env.now() + 1s),
+                fee(1500));
```

Comment:
Why does this transaction need a higher fee?

Reply:
Ditto for all the other 1500 drop fees

Reply:
Didn't check - just copied it from other code. For credentials test it is irrelevant.